cmd/skopeo-sync-gen/skopeo-sync-gen.go

```diff
@@ -0,0 +1,159 @@
+package skopeo_sync_gen
+
+import (
+	"context"
+	"encoding/json"
+	// "fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/konveyor/crane/internal/file"
+	"github.com/konveyor/crane/internal/flags"
+	imagev1 "github.com/openshift/api/image/v1"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	InternalRegistryDefault = "image-registry.openshift-image-registry.svc:5000"
```

Comment:
Sorry if I missed it. Will we be doing anything to consider openshift 3.11, `docker-registry.default.svc:5000`?

Reply:
we'll handle that with the `oc-registry-info` ClusterTask via pipelines. But you can override this default